docs/user/dashboard/lens.asciidoc

```diff
@@ -19,301 +25,316 @@ data-type="inline"
 +
 [role="screenshot"]
 image::images/lens_drag_drop_3.gif[Using drag and drop to reorder]
 
-. Press Space bar to confirm, or to cancel, press Esc.
+. To confirm the action, press Space bar. To cancel, press Esc.
 
 [float]
-[[lens-formulas]]
-==== Use formulas to perform math
+[[configure-the-visualization-components]]
+==== Configure the visualization components
 
-Formulas let you perform math on aggregated data in Lens by typing
-math and quick functions. To access formulas,
-click the *Formula* tab in the dimension editor. Access the complete
-reference for formulas from the help menu.
+Each visualiztion type comes with a set of components that you can configure, such as data labels, legend, axes, and more.
 
-The most common formulas are dividing two values to produce a percent.
-To display accurately, set *Value format* to *Percent*.
+For example, to add data labels to bar and horizontal bar charts, use the *Values* menu.
 
-Filter ratio::
+[role="screenshot"]
+image::images/lens_value_labels_xychart_toggle.png[Lens Bar chart value labels menu]
 
-Use `kql=''` to filter one set of documents and compare it to other documents within the same grouping.
-For example, to see how the error rate changes over time:
-+
-```
-count(kql='response.status_code > 400') / count()
-```
+To add data labels to pie charts, donut charts, or treemaps, use the *Labels* menu.
 
-Week over week:: Use `shift='1w'` to get the value of each grouping from
-the previous week. Time shift should not be used with the *Top values* function.
-+
-```
-percentile(system.network.in.bytes, percentile=99) /
-percentile(system.network.in.bytes, percentile=99, shift='1w')
-```
+[role="screenshot"]
+image::images/lens_value_labels_partition_toggle.png[Lens Pie chart value labels menu]
 
-Percent of total:: Formulas can calculate `overall_sum` for all the groupings,
-which lets you convert each grouping into a percent of total:
-+
-```
-sum(products.base_price) / overall_sum(sum(products.base_price))
-```
+[float]
+[[filter-the-data]]
+==== Filter the data
+
+Apply filters to visualizations directly from the values in the legend. Only *Bar*, *Line and area*, and *Proportion* visualizations support legend filters.
+
+. In the legend, click the field.
+
+. Choose one of the following options:
+
+* *Filter for value* &mdash; Applies a filter that displays only the field data in the visualization.
+
+* *Filter out value* &mdash; Applies a filter that removes the field data from the visualization.
 
 [float]
 [[lens-faq]]
 ==== Frequently asked questions
 
 For the answers to common *Lens* questions, review the following. 
 
-[float]
-[[kql-]]
-===== When should I use the top filter bar, filters function, or "Filter by"?
+[discrete]
+[[kql]]
+.When should I use KQL filters, the Filters function, or the Filter by option?
+[%collapsible]
+====
+To focus on a known set of data for all the visualization results, use the <<semi-structured-search, KQL *Search*>> field.
```

Comment:
This sentence has become more confusing to me, it was important to point out that the *top bar KQL* applies to the whole visualization, and that there are two other places to enter KQL that affect other parts.